src/content/api/merchants.mdx

```diff
@@ -186,6 +186,10 @@ which define the payment methods available for a Payment Request.
     <Property name="location" type="location">
       Physical Location of Merchant. It is highly recommended that you provide this otherwise, users won't be able to find you with our [Merchant Search](#search-merchants) API if they perform a origin + distance query.
     </Property>
+
+    <Property name="type" type="string">
+      Valid values: `physical` or `e-commerce`.
```

Comment:
Do we need to add more information here as to what this value is used for?

Reply:
It is used for reporting purposes. I'd ideally like to avoid including low level explanations such as that in our public docs. Do you agree?

Reply:
I just think if we are expecting the integrator to make a decision then we should provide them enough info to make a decision

Reply:
You probably need to add to the merchant model as well. Maybe that is the place to add more information.